src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.cpp

```diff
@@ -0,0 +1,366 @@
+  switch (coords) {
+    case SchwarzschildCoordinates::Isotropic:
+      return os << "Isotropic";
+    default:
+      ERROR("Unknown SchwarzschildCoordinates");
+  }
+}
+
+template <>
+double Schwarzschild<
+    SchwarzschildCoordinates::Isotropic>::radius_at_horizon() noexcept {
+  return 0.5;
+}
+
+// Conformal metric
+
+template <SchwarzschildCoordinates Coords>
+template <typename DataType>
+tuples::TaggedTuple<Xcts::Tags::ConformalMetric<DataType, 3, Frame::Inertial>>
+Schwarzschild<Coords>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<Xcts::Tags::ConformalMetric<DataType, 3,
+                                           Frame::Inertial>> /*meta*/) const
+    noexcept {
+  auto conformal_metric =
+      make_with_value<tnsr::ii<DataType, 3, Frame::Inertial>>(x, 0.);
+  get<0, 0>(conformal_metric) = 1.;
+  get<1, 1>(conformal_metric) = 1.;
+  get<2, 2>(conformal_metric) = 1.;
+  return {std::move(conformal_metric)};
+}
+
+// Extrinsic curvature trace
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<gr::Tags::TraceExtrinsicCurvature<DataType>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<gr::Tags::TraceExtrinsicCurvature<DataType>> /*meta*/) const
+    noexcept {
+  return {make_with_value<Scalar<DataType>>(x, 0.)};
+}
+
+// Extrinsic curvature trace gradient
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<::Tags::deriv<gr::Tags::TraceExtrinsicCurvature<DataType>,
+                                  tmpl::size_t<3>, Frame::Inertial>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<::Tags::deriv<gr::Tags::TraceExtrinsicCurvature<DataType>,
+                             tmpl::size_t<3>, Frame::Inertial>> /*meta*/) const
+    noexcept {
+  return {make_with_value<tnsr::i<DataType, 3, Frame::Inertial>>(x, 0.)};
+}
+
+// Conformal factor
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<Xcts::Tags::ConformalFactor<DataType>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<Xcts::Tags::ConformalFactor<DataType>> /*meta*/) const noexcept {
+  const DataType r = get(magnitude(x));
+  return {Scalar<DataType>{1. + 0.5 / r}};
+}
+
+// Conformal factor gradient
+
+template <>
+template <typename DataType>
+tuples::TaggedTuple<::Tags::deriv<Xcts::Tags::ConformalFactor<DataType>,
+                                  tmpl::size_t<3>, Frame::Inertial>>
+Schwarzschild<SchwarzschildCoordinates::Isotropic>::variables(
+    const tnsr::I<DataType, 3, Frame::Inertial>& x,
+    tmpl::list<::Tags::deriv<Xcts::Tags::ConformalFactor<DataType>,
+                             tmpl::size_t<3>, Frame::Inertial>> /*meta*/) const
+    noexcept {
+  const DataType r = get(magnitude(x));
```

Comment:
Do you need to allocate r here? Why not instead just cube(get(magnitude(x)) next line instead? Looks like r is otherwise only used for size here